build.rs

```diff
@@ -28,7 +28,9 @@ mod wayland {
 
 fn main() {
     // The script doesn't depend on our code
-    println!("cargo:rerun-if-changed=build.rs:wayland_protocols");
+    println!("cargo:rerun-if-changed=build.rs");
```

Comment:
This line is unnecessary now.

From [cargo reference](https://doc.rust-lang.org/cargo/reference/build-scripts.html#cargorerun-if-changedpath),
> If the build script inherently does not need to re-run under any circumstance, then emitting cargo:rerun-if-changed=build.rs is a simple way to prevent it from being re-run (otherwise, the default if no rerun-if instructions are emitted is to scan the entire package directory for changes). Cargo automatically handles whether or not the script itself needs to be recompiled, and of course the script will be re-run after it has been recompiled. Otherwise, specifying build.rs is redundant and unnecessary.

Reply:
>This build.rs line is unnecessary now. It was useful because the behavior is different between "no rerun-if-changed" and "have any rerun-if-changed".

I decided to keep it since the wayland protocol stuff will be removed anyway, and if it wouldn't be you or me updating it, someone could easily miss to put that `build.rs` line back.

I hope that's clear.